typescript/infra/scripts/funding/fund-keys-from-deployer.ts

```diff
@@ -818,8 +817,8 @@ class ContextFunder {
   ) {
     const l1Chain = L2ToL1[l2Chain];
     const crossChainMessenger = new CrossChainMessenger({
-      l1ChainId: this.multiProvider.getDomainId(l1Chain),
-      l2ChainId: this.multiProvider.getDomainId(l2Chain),
+      l1ChainId: this.multiProvider.getEvmChainId(l1Chain),
```

Comment:
isnt the multiprovider already EVM specific?

Reply:
the multiprovider is evm specific but the function itself is from the chain metadata manager which is not chain specific

it's effectively a shorthand for ensuring we get a number in EVM land